examples/about_dialog/main.rs

```diff
@@ -0,0 +1,44 @@
+use gtk::gdk;
+use gtk::gio;
+use gtk::glib;
+use gtk::prelude::*;
+
+fn main() {
+    let app = gtk::Application::new(None::<&str>, gtk::gio::ApplicationFlags::empty());
+
+    app.connect_activate(|app| {
+        let window = gtk::ApplicationWindow::new(app);
+        window.set_title(Some("Example"));
+
+        let button = gtk::Button::builder()
+            .label("Show About Dialog")
+            .margin_top(24)
+            .margin_bottom(24)
+            .margin_start(24)
+            .margin_end(24)
+            .build();
+
+        let logo_file = gio::File::for_path("./examples/about_dialog/gtk-rs.svg");
```

Comment:
This only works if my cwd is correct, right? Maybe embedding into the binary with a macro is more robust

Reply:
In that case, I don't mind either way :)